README.md

```diff
@@ -0,0 +1,70 @@
+# labkeeper
+
+A tool developed by Ansible for deploying and managing OpenLab CI infrastructures
+
+- License: Apache License, Version 2.0
+- Source: https://github.com/theopenlab/labkeeper
+
+## Description
+
+Labkeeper is a collection of Ansible playbooks and roles used to deploy and manage
+OpenLab CI infrastructures.
+
+This project is initially forked from the [OpenStack Windmill project](https://github.com/openstack/windmill), and
+modified for specific [OpenLab](https://github.com/theopenlab) CI system deployment.
+
+## Steps to deploy a CI system using Labkeeper
+
+1. Choose a deployment mode. Labkeeper now supports:
```

Comment:
I still don't know how to set this deployment mode. It would be good to tell user change which file, which setting to which value to make these modes work.

Reply:
Done